Lib/test/test_dataclasses.py

```diff
@@ -2713,10 +2713,10 @@ class C:
                 self.assertEqual(C(10).x, 10)
 
     def test_classvar_module_level_import(self):
-        from . import dataclass_module_1
-        from . import dataclass_module_1_str
-        from . import dataclass_module_2
-        from . import dataclass_module_2_str
+        from test import dataclass_module_1
+        from test import dataclass_module_1_str
+        from test import dataclass_module_2
+        from test import dataclass_module_2_str
```

Comment:
Are these import changes strictly needed? I'm not at my development environment, and can't easily check. If they are, that's fine. But if not, I'd rather leave them as-is so as to minimize the changes from the 3.6 backport of this code.

Reply:
This change allows to fix this issue. I don't know whether there is other simple way.

Reply:
Okay, then this looks good to me.